lib/importer/src/backend.js

```diff
@@ -544,7 +543,7 @@ exports.SessionPerformMappingJob = (sid, range, mapping, includeErrorRow = false
       // Only if we found something do we validate and map the types
       let mappedRecord = {};
 
-      Object.entries(record).forEach((element) => {
+      Object.entries(recordCells).forEach((element) => {
```

Comment:
Line 565 contains `mapCellValue` which no longer exists. Think might have missed this one.. but github won't let me attach a comment to that line :(